### DISCUSSION TITLE
Who is this charactor?

### DISCUSSION THREAD

**Discussion by malsuke** [moby/moby#48017] [Q&A] [answered]
Somtimes, I wonder who these charactors are. 
I know gopher, docker, tux, and compose. But I don't know others. so could you tell me others name?

![image](https://github.com/moby/moby/assets/153903070/3d0e480f-b393-4273-87ed-b6bf48286224)

**Comment by thaJeztah** [comment, marked as answer]
The turtle is @GordonTheTurtle - Docker's office tortoise.


The school of fish on the left is Swarm, and the Clam was for the open source Docker Registry, which now has a new logo, but still has a shell in it; https://github.com/distribution/distribution

Not sure what the Crab represents, but could be just a cute animal 😄

**Reply by malsuke** [reply]
Waa, I see. 
It was very helpful.
Thank you for letting me know.
